ext/nnstreamer/tensor_filter/tensor_filter_pytorch.cc

```diff
@@ -111,16 +120,43 @@ TorchCore::~TorchCore ()
   g_free (model_path);
 }
 
+/**
+ * @brief	Set the accelerator for the pytorch
+ */
+void
+TorchCore::setAccelerator (const char *accelerators)
+{
+  use_gpu = TRUE;
+  accelerator = parse_accl_hw (accelerators, torch_accl_support);
+  if (accelerator == ACCL_NONE)
+    goto use_gpu_ini;
+  if ((accelerator & (ACCL_CPU | ACCL_DEFAULT | ACCL_AUTO)) != 0)
+    use_gpu = FALSE;
+
+  g_message (" 0 gpu = %d", use_gpu);
+  return;
+
+use_gpu_ini:
+  use_gpu = nnsconf_get_custom_value_bool ("pytorch", "enable_use_gpu",
+      FALSE);
+  if (use_gpu == FALSE) {
+    accelerator = ACCL_NONE;
+  } else {
+    accelerator = ACCL_GPU;
+  }
+  g_message (" 1 gpu = %d", use_gpu);
+}
+
 /**
  * @brief	initialize the object with torch model
  * @return 0 if OK. non-zero if error.
  *        -1 if the model is not loaded.
  */
 int
-TorchCore::init (const GstTensorFilterProperties * prop,
-    const gboolean torch_use_gpu)
+TorchCore::init (const GstTensorFilterProperties * prop)
 {
-  use_gpu = torch_use_gpu;
+  setAccelerator (prop->accl_str);
+  g_message ("gpu = %d, accl = %s", use_gpu, get_accl_hw_str(accelerator));
```

Comment:
How about printing this message only when `DBG` is enabled?

Reply:
@dongju-chae need to set DBG variable for ubuntu build.
Plus this needs to be set in other extensions as well.
Will do this separately.